### PR TITLE
Add the shells for all metadata

### PR DESCRIPTION
**Pull request by owenrumney**
- adding the json for all metadata blocks for all current tfsec checks

- needs refining with CSPM data, but is a starting point to work from

### CODE REVIEW

File: aws/api-gateway/enable-access-logging/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (oranmoshai) [inline aws/api-gateway/enable-access-logging/metadata.json:1]: Can you generate also index file like we did here 5383611fa2f9eaec994bca3eb9b8297e03c1ce1a 
maybe on the way we can auto generate the AVD ID

Comment (owenrumney) [reply aws/api-gateway/enable-access-logging/metadata.json:1]: yes thats no problem. This was a demo PR  - the auto gen of ID will be easy when we know the format

**Comment by oranmoshai** [comment]
LGTM for the shell really helps

File: aws/index.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
+{
```
Comment (oranmoshai) [inline aws/index.json:1]: It seems we need the full path for the metadata file 
This will help the AVD generator

**Comment by oranmoshai** [comment]
LGTM

**Comment by joshlarsen** [comment]
Can we change `custom.urls` to `custom.references` and use this format? It will make rendering in AVD much more useful.

```
[
  {
    "title": "Terraform Access Log Settings",
    "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/apigatewayv2_stage#access_log_settings"
  },
  {
    "title": "AWS Access Logging Setup",
    "url": "https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html"
  }
]
```

**Pull request merged**